Libplanet.Tests/Blockchain/BlockChainTest.cs

```diff
@@ -2049,12 +2049,8 @@ private void CheckIfTxPolicyExceptionHasInnerException()
                 {
                     Assert.NotNull(e.InnerException);
 
-                    // This is visual representation of the test.
-                    // In actual use-case, this debugging lines do
-                    // not show.
                     _logger.Debug("Exception: {Exception}", e.ToString());
                     _logger.Debug("InnerException: {InnerException}", e.InnerException.ToString());
-                    throw e;
                 }
```

Comment:
You probably intended to get rid of only `e`, instead of the whole line, didn't you?

Reply:
I think it really depends on how the original test-writer thought. If the writer wanted to throw that exception anyways, then `throw` would the correct one. However, if the `Debug` code suffices for recording that a `TxPolicyViolationException` took place, then the whole `throw` statement would be unnecessary.

Reply:
I'm the trouble-maker in this situation. I really appreciate that you've caught this. Thanks! 😊

This `TxPolicyViolationException` is used in `BlockPolicy` and `BlockChain<T>.ValidateNextBlockTx()`. 

In one of tests I wrote, this `CheckIfTxPolicyExceptionHasInnerException()`, I wanted to make sure new `BlockChain<T>`, with full dummy policies excepts TxPolicy, throws `TxPolicyViolationException` and only `TxPolicyViolationException`. That's why I've made this test code as followed.

> The contents of the comments are ambiguous; e.g. why should the lines not appear?
> Also, the following Debug lines are quite self-explanatory: they print the exception.

Yes, now that I think about it, that comment was added unnecessarily. Most test cases I've seen were not written with `_logger.Debug(...)` stdout, At that moment I thought comment should be needed somewhat. that's why the comment was added. Also I want to see myself how stacktrace would be looked like. In short, those lines are not that really necessary.

Reply:
Ah, nice! Guess I was a mind-reader then 👀

Reply:
Could you restore `throw;` back for the context of this test and remove `_logger.Debug(...)` lines for now? or Let me know if you have better way to test this. 🙂

Reply:
@colibrishin Hmm... I guess that's the best option for now. I will carry them out.